src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
You should link CSS files in your HTML, not SCSS. Browsers don't understand SCSS syntax. You probably have a compiled CSS file, link that one instead.